src/shared/components/palette/date/write-date-container-field.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { DebugElement } from '@angular/core';
 import { FormGroup, ReactiveFormsModule } from '@angular/forms';
 import { MatDatepickerModule, MatFormFieldModule, MatInputModule } from '@angular/material';
-import { NgxMatDatetimePickerModule, NgxMatNativeDateModule, NgxMatTimepickerModule } from '@angular-material-components/datetime-picker';
+import { NgxMatDatetimePickerModule, NgxMatNativeDateModule, NgxMatTimepickerModule } from '@angular-material-components/datetime-picker/esm5/angular-material-components-datetime-picker.js';
```

Comment:
why is this /esm5/angular-material-components-datetime-picker.js  needed?

Reply:
we need to use the es5 version of this npm package, otherwise IE 11 breaks. i also had to remove the moment-adapter as it doesnt have an es5 version we can use. Although i hate this approach, its the only way in such a short amount of time to keep the datepicker and date components within the toolkit